modules/rostests/rosautotest/main.cpp

```diff
@@ -100,6 +102,22 @@ wmain(int argc, wchar_t* argv[])
             WineTest.Run();
         }
 
+        /* Clear the stringstream */
+        ss.str("");
+        ss.clear();
+
+        /* Show the beginning time */
+        ss << "[ROSAUTOTEST] System uptime at start was " << setprecision(2) << fixed;
+        ss << ((float)TestStartTime/1000) << " seconds" << endl;
```

Comment:
Add spaces around operators:
```suggestion
        ss << ((float)TestStartTime / 1000) << " seconds" << endl;
```

Reply:
I will make this change. Thanks.